### PR TITLE
Add Servicebus

### PR DESCRIPTION
**Pull request by akirakakar**
## Purpose
Added service functionality. Since this is the simplified version, MessageHandler is not registered. 

## Does this introduce a breaking change?

```
[x ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
* Introduced service bus
* Removed Message class because service bus api already had Message class

```
[ ] Bugfix
[ x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by TsuyoshiUshio** [comment]
LGTM. However, one issue remains. It is not about your PR. But DependencyTelemetry requires time. I added a duration however, we need start time. This should be a next PR.

File: JavaSample/src/main/java/processor/ProcessController.java [hunk 1]
```diff
@@ -12,17 +12,25 @@
...
+import com.microsoft.azure.servicebus.QueueClient;
 
 import org.springframework.http.HttpEntity;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class ProcessController {
-    private static final String template = "Parent Request-Id: %s";
+    private static final String connectionString = "{connection string}";
```
Comment (TsuyoshiUshio) [inline JavaSample/src/main/java/processor/ProcessController.java]: Could you make it environment variables? Then customer don't need to touch the code and natural way for production.

File: JavaSample/src/main/java/processor/ProcessController.java [hunk 2]
```diff
@@ -12,17 +12,25 @@
...
     private static final String currentTemplate = "Current Request Id: %s RootId: %s ParentId: %s";
+    private static final String queueName = "{queue name}";
```
Comment (TsuyoshiUshio) [inline JavaSample/src/main/java/processor/ProcessController.java]: You can specify the queue name since this is Sample.

**Comment by TsuyoshiUshio** [comment]
@akirakakar I reviewed and find small fixes.

File: JavaSample/src/main/java/processor/ProcessController.java (hunk 2 above)
Comment (akirakakar) [reply JavaSample/src/main/java/processor/ProcessController.java]: I changed this to environment variable as well.

File: JavaSample/src/main/java/processor/ProcessController.java (hunk 1 above)
Comment (akirakakar) [reply JavaSample/src/main/java/processor/ProcessController.java]: Done.

**Comment by TsuyoshiUshio** [comment]
Thank you, @akirakakar ! LGTM.

**Pull request merged**